core/src/main/java/org/apache/calcite/sql2rel/RelDecorrelator.java

```diff
@@ -442,11 +455,11 @@ public Frame decorrelateRel(Values rel) {
   }
 
   /**
-   * Rewrites a {@link LogicalAggregate}.
+   * Rewrites a {@link Aggregate}.
    *
    * @param rel Aggregate to rewrite
    */
-  public Frame decorrelateRel(LogicalAggregate rel) {
+  public Frame decorrelateRel(Aggregate rel) {
```

Comment:
Here, we would need to keep `decorrelateRel(LogicalAggregate)` for binary compatibility...

Reply:
Done.